modules/issue_tracker/php/NDB_Menu_Filter_Form_my_issue_tracker.class.inc

```diff
@@ -0,0 +1,369 @@
+<?php
+
+/**
+ * This class features the code for the menu portion of the Loris issue
+ * tracker.
+ *
+ * PHP Version 5
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Tools
+ * @author     Caitrin Armstrong <caitrin.armstrong@mail.mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris-Trunk/
+ */
+
+/**
+ * Provides the PHP code for the menu filter for the issue tracker
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Tools
+ * @author     Caitrin Armstrong <caitrin.armstrong@mail.mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris-Trunk/
+ */
+class NDB_Menu_Filter_Form_My_Issue_Tracker extends NDB_Menu_Filter_Form
+{
+    //aka building-table-using-react
+    var $AjaxModule = true;
+
+
+    /**
+     * Set up the variables required by NDB_Menu_Filter class for constructing
+     * a query
+     *
+     * @return null
+     */
+    function _setupVariables()
+    {
+
+        $user =& User::singleton();
+        $db   = Database::singleton();
+
+        $this->query = " FROM issues as i ".
+            "LEFT JOIN candidate c ON (i.candID=c.CandID) " .
+            "LEFT JOIN psc p ON (i.centerID=p.CenterID) ".
+            "LEFT JOIN LorisMenu l on (i.module=l.ID) ".
+            "LEFT JOIN session s ON (i.sessionID = s.ID) ".
+            "LEFT JOIN issues_watching w ON (i.issueID = w.issueID) ".
+            "WHERE i.status != 'closed' ".
+            "AND i.assignee=" . "'". $user->getData('UserID') . "'"; //yup
```

Comment:
this should be quoted with the $db->quote() function instead of hardcoding the quotation marks